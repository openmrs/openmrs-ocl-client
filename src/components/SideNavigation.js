import React from 'react';

const SideNavigation = () => (
  <div className="sidenav" />
);
export default SideNavigation;
