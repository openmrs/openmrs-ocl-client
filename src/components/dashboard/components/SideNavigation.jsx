import React from 'react';
import { Link } from 'react-router-dom';

const SideNavigation = () => (
  <div className="side-nav d-none d-md-block d-lg-block">
    <ul className="nav flex-column">
      <li className="nav-item">
        <Link to="/dashboard" className="nav-link active">
          Sources
        </Link>
        <Link to="/dashboard/concepts" className="nav-link active">
          <i className="fa fa-tags" /> Concepts
        </Link>
      </li>
    </ul>
  </div>
);
export default SideNavigation;
