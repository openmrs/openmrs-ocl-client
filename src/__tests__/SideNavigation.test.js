import React from 'react';
import { mount } from 'enzyme';
import SideNavigation from '../components/SideNavigation';

const wrapper = mount(<SideNavigation />);

describe('SideNavigation Component', () => {
  it('should render without crashing', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
