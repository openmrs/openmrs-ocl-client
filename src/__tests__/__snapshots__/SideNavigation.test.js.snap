// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SideNavigation Component should render without crashing 1`] = `
<SideNavigation>
  <div
    className="sidenav"
  />
</SideNavigation>
`;
